lib/action_cable/railtie.rb

```diff
@@ -0,0 +1,19 @@
+module ActionCable
+  class Railtie < Rails::Railtie
+    config.action_cable = ActiveSupport::OrderedOptions.new
+
+    initializer "action_cable.logger" do
+      ActiveSupport.on_load(:action_cable) { self.logger ||= ::Rails.logger }
+    end
+
+    initializer "action_cable.set_configs" do |app|
+      options = app.config.action_cable
+
+      options.allowed_request_origins ||= "http://localhost:3000" if ::Rails.env.development?
+
+      ActiveSupport.on_load(:action_cable) do
+        options.each { |k,v| self.send("#{k}=", v) }
```

Comment:
Could rely on implicit self and omit it here.

Find it curious we use `send` and not `public_send` in all Rails gems, but it's okay I guess.